tests/30rooms/04messages.pl

```diff
@@ -349,3 +349,55 @@
          })
       });
    };
+
+test "Ephemeral messages received from clients are correctly expired",
+   requires => [ local_user_and_room_fixtures() ],
+
+   do => sub {
+      my ( $user, $room_id ) = @_;
+
+      my $now_ms = int( time() * 1000 );
+      my $filter = '{"types":["m.room.message"]}';
+
+      matrix_send_room_message( $user, $room_id,
+         content => {
+            msgtype                          => "m.text",
+            body                             => "This is a message",
+            "org.matrix.self_destruct_after" => $now_ms + 1000,
+         },
+      )->then( sub {
+          matrix_get_room_messages( $user, $room_id, filter => $filter )
```

Comment:
I'm no fan of the 3-space indentation convention, but let's try to stick to it.

There are more instances below.

Reply:
Ugh Pycharm why